src/complex.js

```diff
@@ -794,17 +787,8 @@ Sk.builtin.complex.prototype.tp$getattr = function (name) {
 
 
 Sk.builtin.complex.prototype.tp$setattr = function (name, value) {
-    if (name != null && (Sk.builtin.checkString(name) || typeof name === "string")) {
-        var _name = name;
-
-        // get javascript string
-        if (Sk.builtin.checkString(name)) {
-            _name = Sk.ffi.remapToJs(name);
-        }
-
-        if (_name === "real" || _name === "imag") {
```

Comment:
do we now support setting `real` or `imag`?

Reply:
only in the same we we did before e.g.
```
>>> complex(1,2).real
1
```
in #1092 we have full support for `real` and `imag` because they are just `getset_descriptors` and so no need to hack `tp$getattr` like we currently do in `complex.js`

in #1092 it's just
```
getsets = {
  real: {
    $get: function () { return new Sk.builtin.float_(this.real) }
  }
...
```
and similar `getsets_descriptors` for `int` and `float`

Reply:
Actually I've made a mistake on tp$setattr here. I'll fix and push.

Reply:
done with latest commit cae22b3